esp/xslt/esdl2monitor.xslt

```diff
@@ -21,6 +21,7 @@
     <xsl:param name="sourceFileName" select="'UNKNOWN'"/>
     <xsl:param name="responseType" select="''"/>
     <xsl:param name="requestType" select="''"/>
+    <xsl:param name="platform" select="'roxie'"/>
```

Comment:
@rpastrana I usually do "ESP" because it is really an acronym E.S.P. 

Roxie letters have an origin story, but I think of it as a name.  So for a parameter "roxie" or in some docs, ect.  "Roxie".  But I guess I could make them consistent.